Bank/src/main/java/com/ark/bank/BankController.java

```diff
@@ -36,6 +37,10 @@ public BankController(String bankId, ICentralBankConnection centralBankConnectio
         this.centralBankConnection = centralBankConnection;
     }
 
+    public void connectToBankDatabase(String id){
+        this.BankDatabaseConnection = new DatabaseController(id);
```

Comment:
Makr onnectToDatabase a boolean method. Check is string not null or empty. rename id to bankId (we use that everywhere)